server/http_handler_test.go

```diff
@@ -873,7 +876,7 @@ func TestGetSchema(t *testing.T) {
 	err = decoder.Decode(&lt)
 	require.NoError(t, err)
 	require.NoError(t, resp.Body.Close())
-	require.Greater(t, len(lt), 0)
+	require.Equal(t, len(lt), 3)
```

Comment:
Maybe greater than 2 is enough? Otherwise it is easy to break if someone add tables to `tidb` in test.